libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.model.ts

```diff
@@ -22,8 +22,15 @@ export interface ToggleData {
 
 
   /**
-   * The custom color for the toggle.
+   * The custom color for the toggle when activated.
    * @Optional
    */
   color?: string;
+
+
+    /**
+   * The custom color for the toggle when unactivated.
+   * @Optional
+   */
+  uncheckedColor?: string;
```

Comment:
Please include access modifier for all properties

Reply:
Interfaces define public contracts, so all members is public and no modifier is required